site-src/geps/gep-922.md

```diff
@@ -0,0 +1,163 @@
+# GEP-922: Gateway API Versioning
+
+* Issue: [#922](https://github.com/kubernetes-sigs/gateway-api/issues/922)
+* Status: Implementable
+
+## TLDR
+Each Gateway API release will be represented by a bundle version that represents
+that specific combination of CRDs, API versions, and validating webhook. To
+enable experimental fields, a new experimental set of CRDs will be included with
+future releases.
+
```

Comment:
We should state what we are calling this, to avoid any confusion.

"Stable" vs "Experimental" CRD version.

Should also probably say the API version (e.g. v1alphaX) does not represent the right model for CRD versioning.

Reply:
Updated to make this a bit more clear